src/components/Preloader/preloader.css

```diff
@@ -0,0 +1,7 @@
+.preloader {
+	background-color: #F5F6F7;
```

Comment:
A piece of advice for any forms:


**Might be better**



You can make a custom hook for inputs handling:

```js
export function useForm(inputValues) {
  const [values, setValues] = useState(inputValues);

  const handleChange = (event) => {
    const {value, name} = event.target;
    setValues({...values, [name]: value});
  };
  return {values, handleChange, setValues};
}

```

This way you can easily handle any form in your project